src/external_data_ifaces_impl.cpp

```diff
@@ -41,4 +42,48 @@ sdbusplus::async::task<> ExternalDataIFacesImpl::fetchBMCRedundancyMgrProps()
     co_return;
 }
 
+// NOLINTNEXTLINE
+sdbusplus::async::task<> ExternalDataIFacesImpl::fetchSiblingBmcIP()
+{
+    // TODO: Currently, the IP addresses for both BMCs are hardcoded.
+    // Once the network DBus exposes the IPs, update the logic to retrieve
+    // them dynamically from Dbus.
+
+    // TODO: Handle the exception and exit gracefully, as the data sync relies
+    //      heavily on these DBus properties and cannot function effectively
+    //      without them.
+
+    const data_sync::ext_data::SiblingBmcIP siblingIp0 = "10.0.2.100";
+    const data_sync::ext_data::SiblingBmcIP siblingIp1 = "10.2.2.100";
+
+    using siblingBmc = sdbusplus::client::xyz::openbmc_project::state::bmc::
```

Comment:
I understand this is a workaround, but here’s a suggestion for you to consider for future commits:

1. You could use a `using` alias for `sdbusplus::common::xyz::openbmc_project::state::bmc::redundancy::Sibling` to improve readability.
2. The naming for the `proxy object` isn’t very clear.

Here’s an example of how I would structure it:
```
using SiblingBMCMgr = sdbusplus::client::xyz::openbmc_project::state::bmc::redundancy::Sibling<>;
using SiblingBMC = sdbusplus::common::xyz::openbmc_project::state::bmc::redundancy::Sibling;

auto siblingBMCMgr = SiblingBMCMgr(_ctx)
                       .service(SiblingBMC::interface)
                       .path(siblingBMCInstancePath);

auto siblingBMCPosition = co_await siblingBMCMgr.bmc_position();
```